data/brands/amenity/cafe.json

```diff
@@ -2789,9 +2807,10 @@
     },
     {
       "displayName": "星乃珈琲店",
-      "id": "hoshinocoffee-0cf217",
-      "locationSet": {"include": ["jp"]},
+      "id": "hoshinocoffee-7f2f5a",
+      "locationSet": {"include": ["jp", "tw"]},
       "tags": {
+        "alt_name:zh": "星乃咖啡店",
```

Comment:
Why not use ```name:zh-Hans``` because this is written in Simplified Chinese?

Reply:
And write a ```name:zh-Hant=星乃珈琲店```  at the same time

Reply:
Yes and no, actually `星乃珈琲店` is a Japanese Kanji word, 
it would be same Chinese characters `星乃咖啡店` as in both Traditional and Simplified Chinese. 
However, I kept the `name=星乃珈琲店` w/ `alt_name:zh=星乃咖啡店` since they use the same branding in TW.

`星乃珈琲店` 其實是日本漢字, 簡中/䌓中的字符實際上一樣是 `星乃咖啡店`, 
但它們上陸時候沿用所以才會選擇用 `name=星乃珈琲店` + `alt_name:zh=星乃咖啡店` 的組合